ipatests/test_webui/test_automember.py

```diff
@@ -296,11 +296,11 @@ def test_rebuild_membership_hosts(self):
         # Remove host from hostgroup
         self.delete_record(host1)
 
-        # Assert that host is not a member of hostgroup
+        # Assert that host was re-added to the group
+        # See https://www.port389.org/docs/389ds/design/automember-postop-modify-design.html
```

Comment:
Please add a comment stating that this behavior is expected with the plugin default setting: the entry `cn=Auto Membership Plugin,cn=plugins,cn=config` has a default value `autoMemberProcessModifyOps: on` which means that the plugin gets triggered and re-adds the entry.

Reply:
Fixed